ecs-agent/tcs/handler/handler_test.go

```diff
@@ -441,9 +440,6 @@ func TestClientReconnectsAfterInactiveTimeout(t *testing.T) {
 	// it would continue to reconnect and test will be in forever loop.
 	assert.False(t, websocket.IsCloseError(err, websocket.CloseAbnormalClosure),
 		"Read from closed connection should produce an io.EOF error")
-
-	assert.Equal(t, err.Error(), context.DeadlineExceeded.Error(), "Context deadline exceeded error expected.")
```

Comment:
nit (non-blocking): Just curious, is there a specific reason why this assert statement was removed from this test?

Reply:
Yes, the test was flaky. The return code differs based on when the ctx is cancelled. If the context is cancelled when we are in client.consumeMessages, it results in non-nil error. But when the context is cancelled in starttelemetrysession(), it results in a nil error. Another way to test would be to test for either of the condition to be true. Let me try that out and i can re-add both the conditions in a later PR.